.github/workflows/integration-test.yml

```diff
@@ -34,12 +35,36 @@ jobs:
           echo "PLUGIN_VERSION=$plugin_version" >> $GITHUB_ENV
         shell: bash
 
-      - name: Download security plugin and create setup scripts
+      - name: Download security plugin and create setup scripts for remote cluster
+        uses: ./.github/actions/download-plugin
+        with:
+          opensearch-version: ${{ matrix.datasource_version }}
+          plugin-name: ${{ env.PLUGIN_NAME }}
+          download-location: ${{env.PLUGIN_NAME}}-${{matrix.datasource_version}}
+          plugin-version: ${{matrix.datasource_version}}.0
+
+      - name: Download security plugin and create setup scripts for local cluster
         uses: ./.github/actions/download-plugin
         with:
           opensearch-version: ${{ env.OPENSEARCH_VERSION }}
           plugin-name: ${{ env.PLUGIN_NAME }}
+          download-location: ${{env.PLUGIN_NAME}}
           plugin-version: ${{ env.PLUGIN_VERSION }}
+      
+      - name: Run Opensearch with A Single Plugin Remote Cluster
+        uses: derek-ho/start-opensearch@graceful-t
```

Comment:
Looks like `graceful-t` needs to be published?

Reply:
Good catch! Thanks! I was iterating fast on a individual branch but I published it to v4 and consumed it in this commit: [b6fd710](https://github.com/opensearch-project/security-dashboards-plugin/pull/1888/commits/b6fd71079d981650e8bb79c111be17ee6dc8347c).